src/actinia_core/rest/ephemeral_processing_with_export.py

```diff
@@ -294,6 +294,51 @@ def _export_raster(self, raster_name,
 
         return file_name, output_path
 
+    def _export_strds(self, strds_name, format="GTiff"):
+        """Export a specific strds layer with t.rast.export.
+
+        The result is stored in a temporary directory
+        that is located in the temporary grass database.
+
+        Args:
+            strds_name (str): The name of the strds layer
+            format (str): GTiff (only option)
+
+        Returns:
+            tuple: A tuple (file_name, output_path)
+
+        """
+        suffix = ".tar.bzip2"
```

Comment:
bzip2 works really nice with text and integer data, but not with already compressed data, and here the tifs are already compressed with COMPRESS=LZW. Generally, compressors struggle to compress already compressed data, there is usually little to no size reduction, and it takes quite long. Therefore I would recommend `t.rast.export compression=no`

Reply:
Some quick test shows that both bzip2 and gzip do reduce file size of a tar with LZW compressed GeoTIFFs of Float64. Example is the process chain [above](https://github.com/mundialis/actinia_core/pull/236#issue-715097084 with `compression=no`, then using bzip2 and gzip on the tar file:
```
 122880 Aug 18 19:42 lst_test.tar
  77758 Aug 18 19:42 lst_test.tar.bz2
  76299 Aug 18 19:42 lst_test.tar.gz
```
Thus I recommend `compression=gzip`. gzip might in theory not compress as heavily as bzip2, but if further compression fails to reduce file size, it is faster than bzip2, thus it seems to be a reasonable compromise.

Reply:
Thanks for the test! I changed it to suggested `gzip`.